pkg/internal/checkup/trex/client_test.go

```diff
@@ -0,0 +1,277 @@
+/*
+ * This file is part of the kiagnose project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2023 Red Hat, Inc.
+ *
+ */
+
+package trex_test
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+	"time"
+
+	expect "github.com/google/goexpect"
+
+	assert "github.com/stretchr/testify/require"
+
+	"github.com/kiagnose/kubevirt-dpdk-checkup/pkg/internal/checkup/trex"
+)
+
+const (
+	trafficGeneratorPacketsPerSecond = "1m"
+	testDuration                     = time.Second
+	verbosePrintsEnabled             = false
+
+	portIdx = trex.SourcePort
+)
+
+func TestGetPortStatsSuccess(t *testing.T) {
+	t.Run("when GetPortStats succeed", func(t *testing.T) {
```

Comment:
Please consider not using a sub-test if there is only one.

Reply:
DONE